script.js

```diff
@@ -13,3 +13,212 @@ navLinks.forEach((link) => {
+  },
+  {
+    imgSrc: './Asset/oval.svg',
+    title: 'Tonic',
+    canopy: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    languages: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    imgSrc: './Asset/prof.svg',
+    title: 'Multi-Post Stories',
+    canopy: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    languages: ['HTML', 'CSS', 'JavaScript'],
+  },
+];
+
+const popupContainer = document.createElement('div');
+popupContainer.classList.add('popup-container');
+document.body.appendChild(popupContainer);
+
+function popup(param) {
+  if (param < 0 || param >= popy.length) {
+    // Handle the out-of-bounds case here
+    return param;
+  }
+
+  const seeprobtns = document.querySelectorAll('.seepro');
+  seeprobtns.forEach((btn, index) => {
+    btn.addEventListener('click', () => {
+      const html = popup(index);
+      popupContainer.innerHTML = html;
+      popupContainer.style.display = 'block';
+
+      const closeBtn = popupContainer.querySelector('.close');
+      closeBtn.addEventListener('click', () => {
+        popupContainer.style.display = 'none';
+      });
+    });
+  });
+
+  popupContainer.addEventListener('click', (event) => {
+    if (event.target === popupContainer) {
+      popupContainer.style.display = 'none';
+    }
+  });
+
+  const html = `
+    <div class="pone">
+      <span class="close">&times;</span>
+      <h1>${popy[param].title}</h1>
+      <ul class="po">
+        <li>
+          <span class="canop">${popy[param].canopy}</span>
+          <span class="role">${popy[param].role}</span>
+          <span class="year">${popy[param].year}</span>
+        </li>
+      </ul>
+      <img src="${popy[param].imgSrc}">
+      </img>
+      <p>${popy[param].description}</p>
+      <ul class="hcj">
+        ${popy[param].languages
+    .map((lang) => `<li class="pop_language">${lang}</li>`)
+    .join('')}
+      </ul>
+      <button class="popnewa">See live</button>
+      <button class="popnewb">See source</button>
+    </div>
+  `;
+
+  return html;
+}
+
+const seeprobtn = document.querySelectorAll('.seepro');
+
+for (let i = 0; i < seeprobtn.length; i += 1) {
+  seeprobtn[i].addEventListener('click', () => {
```

Comment:
Kindly take note that the functionalities of your popup aren't working properly🥲 and your entire page is overflowing which I believe is a bug that was introduced in this milestone since it still looks great on your live demo😉 Kindly take a look at the video below and ensure when a user clicks the button, the popup menu is opened just as[ required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) in this milestone and do well to fix the issues with the overflow as this can lead to a frustrating user experience🥲👍


https://user-images.githubusercontent.com/64643995/232013681-2d533ee9-920a-4a86-b95e-467c51ef2377.mp4